.github/workflows/kb_sdk-actions.yml

```diff
@@ -61,8 +61,6 @@ jobs:
           LANGUAGE_TOKEN: ${{ matrix.language }}
```

Comment:
A general concern I have is that this makes it easier for users to mismatch their token and environment, which happens all the time even with a test file where both token and environment are in the same context. Now the token will be in an env  var and the environment in the test.cfg file.

Reply:
There's already a barrier to entry for using this code, which is that users have to set an environment variable; not all users will know how to do this, and will probably be put off by that requirement.

For users who are comfortable with it, having dev tokens defined as env vars in their shell can make kbase life quicker and easier; e.g. I have `KBASE_TEST_TOKEN_APPDEV`, `KBASE_TEST_TOKEN_CI`, and `KBASE_TEST_TOKEN` set up in my bashrc, and I alter the last of those to whichever value is appropriate for what I'm doing. It's also very convenient for automated testing via GitHub actions.

Ideally it would be possible to set the server to test against (appdev / CI / etc.) via env var, too, to facilitate automated testing against different servers without having to resort to hacky `sed` statements to edit files in place.

Reply:
Hmm, if very few users are going to figure out how to use this feature, is it worth the maintenance burden?

Reply:
The incremental amount of code is quite small, and it cleans up how things runs. I support it.

Reply:
This doesn't add another other opportunities for user confusion about their environments, so while that is a general concern, it is applies to all such configurations.

Reply:
> The incremental amount of code is quite small

Note there's more code for this change in other PRs.

Reply:
> This doesn't add another other opportunities for user confusion about their environments

I don't think this is true. As I pointed out in the OC, it's a lot easier to figure out when you have a mismatch when the token and env specification live in the same place.